livingdoc-converters/src/main/kotlin/org/livingdoc/converters/collection/AbstractCollectionConverter.kt

```diff
@@ -0,0 +1,53 @@
+package org.livingdoc.converters.collection
+
+import org.livingdoc.api.conversion.ConversionException
+import org.livingdoc.api.conversion.TypeConverter
+import org.livingdoc.converters.TypeConverters
+import java.lang.reflect.*
+
+abstract class AbstractCollectionConverter<T : Collection<Any>> : TypeConverter<T> {
+
+    val defaultSeparator = ","
+
+    @Throws(ConversionException::class)
+    override fun convert(value: String, element: AnnotatedElement?, documentClass: Class<*>?): T {
+        val tokens: List<String> = tokenize(value)
+        val convertedParams = convertToTypedParameters(element, documentClass, tokens)
+
+        return convertToTarget(convertedParams)
+    }
+
+    private fun tokenize(value: String): List<String> {
+        return value.split(delimiters = defaultSeparator).map { it.trim() }
+    }
+
+    private fun convertToTypedParameters(element: AnnotatedElement?, documentClass: Class<*>?, tokenized: List<String>): List<Any> {
+
+        val paramTypeConverter = when (element) {
+            is Field -> { findTypeConverterForTypedParam(element, documentClass) }
+            is Parameter -> { findTypeConverterForTypedParam(element, documentClass) }
+            else -> null
```

Comment:
Easily done by the way with Kotlin's built-in `error()`: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/error.html